docs/data-tools/spack.md

```diff
@@ -0,0 +1,412 @@
+# Spack
+
+[Spack](https://github.com/spack/spack) is a package manager, a tool to assist
+with building and installing software as well as determining what dependencies
+are required and installing those. It was originally designed for use on HPC
+clusters, where several variations of a given package may be installed alongside
+one another for different use cases -- for example different versions, built
+with different compilers, using MPI or hybrid MPI+OpenMP. Spack is principally
+written in Python but has a component written in Answer Set Programming (ASP)
+which is used to determine the required dependencies for a given package
+installation.
+
+Users are welcome to install Spack themselves in their own directories, but we
+are making an experimental installation tailored for ARCHER2 available
+centrally. This page provides documentation on how to activate and install
+packages using the central installation on ARCHER2. For more in-depth
+information on using Spack itself please see the [developers'
+documentation](https://spack.readthedocs.io/en/latest/).
+
+!!! important
+    As ARCHER2's central Spack installation is still in an experimental stage
+    please be aware that we cannot guarantee that it will work with full
+    functionality and we may not be able to provide support.
+
+## Activating Spack
+
+As it is still in an experimental stage, the Spack module is not made available
+to users by default. The development modules must firstly be enabled by running:
+    
+    auser@ln01:~> module use /work/y07/shared/archer2-lmod/apps/dev
```

Comment:
Instead of this instruction, can we uspdate the `other-software` module to add the right path? Then this part would become `module load other-software` and users would get the usual warning about non-supported software.

Reply:
Using `other-software` is a good idea. I've moved the Spack modules into the `archer2-lmod/others/core` path instead of adding the `dev` path to the module, as there are some other developmental modules in there that I'm not sure we want to make available even through `other-software.